src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmployeeController.php

```diff
@@ -476,6 +478,20 @@ protected function getEmployeeFormHandler()
     protected function getErrorMessages(Exception $e)
     {
         return [
+            PasswordConstraintException::class => [
+                PasswordConstraintException::INVALID_LENGTH => $this->trans(
+                    'Password length must be between %min% and %max% characters.',
+                    'Admin.Notifications.Error',
+                    [
+                        '%min%' => (int) $this->get('prestashop.adapter.legacy.configuration')->get(PasswordPolicyConfiguration::CONFIGURATION_MINIMUM_LENGTH),
+                        '%max%' => (int) $this->get('prestashop.adapter.legacy.configuration')->get(PasswordPolicyConfiguration::CONFIGURATION_MAXIMUM_LENGTH),
+                    ]                
+                ),
+                PasswordConstraintException::WEAK_PASSWORD => $this->trans(
+                    'The password doesn\'t meet required strength requirement.',
```

Comment:
Why don't reuse `$tooWeakMessage` defined in `Password.php`? (you have other cases like this one in your PR)

Reply:
Plus _"required strength requirement"_ is just weird :p